src/locales/en-US/list.ftl

```diff
@@ -191,13 +198,12 @@ simply-sign-in-other-device = Simply sign in to your { -fxaccount-brand-name } o
 download-mobile = Download the mobile app
 download-ios-android = { -product-full-name } is available on both iOS and Android. <learnmore>Click here</learnmore> to learn more and to send a link to your phone to download the app.
 before-access = Before you can access your logins on another device, you will need to connect a { -fxaccount-brand-name }.
-connect-a-firefox-account = Connect a { -product-full-name }
-connect-a-firefox-account-complete = Connect a { -product-full-name } (complete)
+connect-a-firefox-account = Connect a { -fxaccount-brand-name }
+# Displayed after the connection has completed
```

Comment:
I'm still not sure if this comment is correct.

Reply:
It is; the string in the modal changes after you have signed in:

before:


<img width="1067" alt="Screen Shot 2019-04-29 at 10 01 02 AM" src="https://user-images.githubusercontent.com/96396/57386275-e2602500-7168-11e9-88f1-2bf9cb44bda9.png">


after:

<img width="1067" alt="Screen Shot 2019-04-29 at 10 03 28 AM" src="https://user-images.githubusercontent.com/96396/57386274-e2602500-7168-11e9-9700-13e0d9b2dded.png">